CMakeLists.txt

```diff
@@ -0,0 +1,33 @@
+cmake_minimum_required(VERSION 3.25.0 FATAL_ERROR)
```

Comment:
hmm, is 3.25 really required here? the newest one which I have in any dev env that I'm actively using is 3.22.1 (although sfos 5.0 got cmake 3.27, from that pov 3.25 isn't really a showstopper, I'm just curious)